userguide/system.rst

```diff
@@ -2375,8 +2375,38 @@ To generate a support ticket, fill in the fields:
   diagnosing a performance bottleneck.
 
 * **Environment** is a drop-down menu to indicate the role of the
-  affected system. Choices are *Production*, *Staging*, *Test*,
-  *Prototyping*, or *Initial Deployment/Setup*.
+  affected system.
+
+  .. tabularcolumns:: |>{\RaggedRight}p{\dimexpr 0.20\linewidth-2\tabcolsep}
+                      |>{\RaggedRight}p{\dimexpr 0.16\linewidth-2\tabcolsep}
+
+.. _environment options:
+
+.. table:: Environment Options
+      :class: longtable
+
+   +---------------------+----------------------------------------------------------+
+   | Environment         | Description                                              |
+   +=====================+==========================================================+
+   | Production          | The system is being used for production. Data loss       |
+   |                     | affects customers directly.                              |
```

Comment:
The sentence for the "Production" environment reads a little harshly and implies there is data loss when there is a production outage. My recommendation would be to say something like `The system is functional and being used in a production environment.`